pkg/object/bos.go

```diff
@@ -38,8 +38,9 @@ const bosDefaultRegion = "bj"
 
 type bosclient struct {
 	DefaultObjectStorage
-	bucket string
-	c      *bos.Client
+	bucket       string
+	storageClass string
```

Comment:
sc for sort

Reply:
for short  `storageClass string`  -> `sc string`  ?